groups/groups.yaml

```diff
@@ -446,6 +447,18 @@ groups:
       - stefan.schimanski@gmail.com
       - thockin@google.com
 
+  - email-id: k8s-infra-rbac-release-triageparty@kubernetes.io
+    name: k8s-infra-rbac-release-triageparty
+    description: |-
+      ACL for Bug Triage Release Team
+    settings:
+      ReconcileMembers: "true"
+      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW"
+    members:
```

Comment:
@justaugustus @markyjackson-taulia I'll need your help to figure out people that need to be added.